clang/test/CodeGen/ppc-register-names.c

```diff
@@ -0,0 +1,14 @@
+// REQUIRES: powerpc-registered-target
+// RUN: %clang -target powerpc64le -c %s -mllvm -stop-after=finalize-isel -o - | \
+// RUN:   FileCheck %s
+// RUN: %clang -target powerpc64 -c %s -mllvm -stop-after=finalize-isel -o - | \
+// RUN:   FileCheck %s
```

Comment:
Could we use `%clang_cc1`?

Reply:
Looks `%clang_cc` doesn't accept `-mllvm`.

Reply:
Why `-c` here? I think it will work with `-S`.

Reply:
`-mllvm -stop-after=finalize-isel` should make `-c` and `-S` no difference, I think.

Reply:
That's because `clang -cc1` doesn't recognize `-c`, instead, it reads `-emit-obj`.